Sources/SmileID/Classes/DocumentVerification/Model/OrchestratedDocumentVerificationViewModel.swift

```diff
@@ -121,17 +121,38 @@ internal class IOrchestratedDocumentVerificationViewModel<T, U: JobResult>: Obse
 
         let zip: Data
         do {
-            let savedFiles = try LocalStorage.saveDocumentImages(
-                front: documentFrontFile,
-                back: documentBackFile,
+            var allFiles = [URL]()
+            let frontDocumentUrl = try LocalStorage.createDocumentFile(jobId: jobId, document: documentFrontFile)
+            allFiles.append(frontDocumentUrl)
```

Comment:
You could bootstrap this with the `selfieFile` and `frontDocumentUrl`
```suggestion
            let frontDocumentUrl = try LocalStorage.createDocumentFile(
                jobId: jobId,
                document: documentFrontFile
            )
            var allFiles = [selfieFile, frontDocumentUrl]
```

Reply:
I was actually thinking of removing this class entirely `DocumentCaptureResultStore` and clean up that function, but in a PR down the line

Reply:
That would be great